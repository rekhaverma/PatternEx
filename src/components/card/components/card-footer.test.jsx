import React from 'react';
import CardFooter from './card-footer.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<CardFooter />', () => {
  it('should match with snapshot', () => {
    const props = {
      content: 'footer content',
    };
    const wrapper = shallow(<CardFooter {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
