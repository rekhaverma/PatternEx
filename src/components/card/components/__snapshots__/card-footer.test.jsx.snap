// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardFooter /> should match with snapshot 1`] = `
<div
  className="card__footer"
>
  footer content
</div>
`;
